addons/website_payment/controllers/portal.py

```diff
@@ -30,6 +30,7 @@ def donation_pay(self, **kwargs):
         if request.env.user._is_public():
             kwargs['partner_id'] = request.env.user.partner_id.id
             kwargs['access_token'] = payment_utils.generate_access_token(kwargs['partner_id'], kwargs['amount'], kwargs['currency_id'])
+            kwargs['hide_tokenization'] = True
```

Comment:
I'd reuse the existing `is_donation` kwarg that is already used everywhere to determine if the current flow is a donation-specific flow or not. This would also avoid adding yet another something-something-tokenization key that could be confused with the others.